vllm/config.py

```diff
@@ -103,7 +103,8 @@ def __init__(
         if os.environ.get("VLLM_USE_MODELSCOPE", "False").lower() == "true":
             # download model from ModelScope hub,
             # lazy import so that modelscope is not required for normal use.
-            from modelscope.hub.snapshot_download import snapshot_download  # pylint: disable=C
+            from modelscope.hub.snapshot_download import \
+                snapshot_download  # pylint: disable=C
```

Comment:
Can we somehow config isort to use `()` for line continuation?
```suggestion
            from modelscope.hub.snapshot_download import (
                snapshot_download)  # pylint: disable=C
```

Reply:
Yep. Will fix it by EoD today

Reply:
fixed